src/components/global/Footer.jsx

```diff
@@ -1,8 +1,195 @@
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { useTheme } from '@mui/material/styles';
+// Other Library Imports
+import dayjs from 'dayjs';
+import { FaDiscord, FaGithub, FaLinkedin, FaMeetup } from 'react-icons/fa6';
+
+// Legal links
+const legalLinks = [
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'About'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'Code of Conduct'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'Contact Us'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    ml: 0.5,
+    text: `©${dayjs().year()}`,
+    title: 'CODE PDX'
+  }
+];
+
+// Social media links
+const socialMediaLinks = [
+  {
+    href: 'https://github.com/codeforpdx/',
+    icon: <FaGithub size={45} />
+  },
+  {
+    href: 'https://www.linkedin.com/company/code-pdx/',
+    icon: <FaLinkedin size={45} />
+  },
+  {
+    href: 'https://www.meetup.com/Code-for-PDX/',
+    icon: <FaMeetup size={45} />
+  },
+  {
+    href: 'https://discord.gg/x6b573et',
+    icon: <FaDiscord size={45} />
+  }
+];
+
+// renders socialMediaLinks for placement in stack below
+const renderSocialLinks = socialMediaLinks.map(({ href, icon }) => (
+  <Link
+    key={href}
+    href={href}
+    target="_blank"
+    rel="noopener"
+    color="#000"
+    sx={{
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      '&:hover': {
+        color: 'secondary.main'
+      }
+    }}
+  >
+    {icon}
+  </Link>
+));
+
+const socialBlobStyle = {
+  backgroundImage: 'url(/assets/socialsBlob.svg)',
+  backgroundSize: 'cover',
+  backgroundRepeat: 'no-repeat',
+  backgroundPosition: 'center',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflow: 'visible',
+  minWidth: '55%',
+  minHeight: '200px'
+};
+
+const logoBlobStyle = {
+  backgroundImage: 'url(/assets/logoBlob.svg)',
+  backgroundRepeat: 'no-repeat',
+  backgroundPosition: 'center',
+  overflow: 'visible'
+};
+
 const Footer = () => {
-  // any logic needed
-  // return (
-  // all the JSX (looks like HTML) to render
-  // )
+  const theme = useTheme();
+
+  // maps legalLinks used for placement in stack below
+  const renderLegalLinks = legalLinks.map((link, index) => (
+    <Box key={link.title + index}>
+      <Typography
+        key={link.title}
+        variant="body2"
+        color="tertiary.main"
+        mr={'10px'}
+        sx={{
+          display: 'flex',
+          justifyContent: 'flex-end',
+          [theme.breakpoints.down('sm')]: {
+            p: '20px'
```

Comment:
This might be too much padding imo. Maybe half as much, or alternatively we could figure additional styling and make the the mobile version of legal links more distinct. For instance, make the font larger, change from `flex-end` to `center`, etc.

Reply:
Agree on padding, cut in half.

Reply:
halved it